analysis/interactive.Rmd

```diff
@@ -4,21 +4,256 @@ output:
+
+With `iSEE`, this can be achieved at any time by clicking on the "Display panel settings" button under the "Diagnostics" tab in the top right corner, copying the displayed code into an R script, and sharing this chunk of code with collaborators.
+However, note that they will also need to have access to a copy of the original `sce` object used for the analysis.
+In that setup, executing the code in the script and launching `iSEE` with the following command will open an instance mimicking what was shown on the screen when the "Display panel settings" button was clicked.
+
+It is important to note that all `iSEE` applications remain interactive irrespective of their initial configuration; only the initial layout and state of the application is modified from the default setup.
+
+### Dissemination of analysis results {#dissemination}
+
+Typically, results from single-cell data are disseminated as static figures in published papers and reports.
+However, those are unlikely to capture the full information content of a complex scRNA-seq data set, and do not allow further exploration by the reader.
+To complement this traditional mode of dissemination, it is becoming increasingly common to deploy web-based interactive data browsers, allowing interested readers to investigate their respective hypotheses using the published data.
+
+With `iSEE`, such interactive exploration is facilitated by small guided tours - step-by-step walkthroughs of the different panels with pointers to facilitate their interpretation, from the authors' point of view.
+At any time, the viewer is still free to leave the interactive tour and explore the data from their own perspective.
+All that is needed to add a tour to an `iSEE` instance is a data frame with two columns named "element" and "intro"; the first column declares the UI element to highlight in each step of the tour, and the second one contains the text to display at that step.
+This data frame must then be provided to the `iSEE()` function via the `tour` argument.
+Examples of tours can be found in https://github.com/LTLA/iSEE2018.
+
+## Reproducibility
+
+Although this chapter focuses on interactive exploration of data, we note that it is important to retain reproducibility, and to have a record of how each figure was generated.
+With `iSEE`, this information is readily available via the "Export R code" button under the "Diagnostics" tab in the top-right corner of the GUI.
+At any time, copying the code displayed in the modal window and executing it in the R session from which the `iSEE` app was launched exactly reproduces all plots currently displayed in the GUI.
+
+## Additional resources
+
+- Publication (F1000Research, 2018): https://f1000research.com/articles/7-741/v1
+- Source code: https://github.com/csoneson/iSEE
```

Comment:
We don't need these two things above. Just talk about the deployed examples.

Reply:
Done in e1f154de

Reply:
You might want to expand a bit more on what you mean by deployed instances. Just have a few sentences outside of a list, and then just list out the ones we have.